apps/frontend/src/renderer/components/GlobalDownloadIndicator.tsx

```
@@ -0,0 +1,149 @@
+
+  return (
+    <div className="fixed bottom-4 right-4 z-50 max-w-sm">
+      <div className="rounded-lg border border-border bg-card shadow-lg overflow-hidden">
+        {/* Header */}
+        <div
+          className={cn(
+            'flex items-center justify-between px-3 py-2 cursor-pointer',
+            hasActive ? 'bg-primary/10' : 'bg-muted/50'
+          )}
+          onClick={() => setIsExpanded(!isExpanded)}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)